webapp/platform/client/src/client4.ts

```diff
@@ -838,6 +838,28 @@ export default class Client4 {
         );
     };
 
+    loginPasswordless = (token: string) => {
```

Comment:
Nit: this should be `loginWithMagicLink` if we're calling the feature "magic link"